lib/Sabberworm/CSS/CSSList/CSSList.php

```diff
@@ -111,14 +112,18 @@ private static function parseAtRule(ParserState $oParserState) {
 			$oParserState->consumeWhiteSpace();
 			$sMediaQuery = null;
 			if (!$oParserState->comes(';')) {
-				$sMediaQuery = $oParserState->consumeUntil(';');
+				$sMediaQuery = trim($oParserState->consumeUntil(array(';', ParserState::EOF)));
+			}
+			if (!$oParserState->isEnd()) {
+				$oParserState->consume(';');
 			}
```

Comment:
I think you could remove these lines if you set `consumeEnd` to `true` in the call to `consumeUntil` (at least it’s supposed to work this way).